toolkit/tools/pkg/imagecustomizerlib/extractpartitions.go

```diff
@@ -34,15 +55,36 @@ func extractPartitions(imageLoopDevice string, outDir string, basename string, p
 			case "raw":
 				// Do nothing for "raw" case.
 			case "raw-zst":
-				partitionFilepath, err = compressWithZstd(partitionFilepath)
+				partitionRawFilePath := partitionFilePath
```

Comment:
nit: Filepath* for consistency (in both variables)

Reply:
please keep this consistent everywhere in the file

Reply:
Addressed this comment.